wordpress-mobile/WordPress-iOS
# Feature/placed pages search

This PR includes the search for the Properly Placed Pages task

![ppp-search](https://user-images.githubusercontent.com/912252/46961763-6b165700-d099-11e8-872e-959a44dab6ba.jpg)

Now the search is one and not filter related. It also displays all the blog pages grouped by the page status.

![ppp-search-parent](https://user-images.githubusercontent.com/912252/46961813-85503500-d099-11e8-89df-ca458f95de81.jpg)

There is a search into the *set parent view controller* (thanks @jklausa for the hint) to to fetch the required page locally.

**To test:**
- Search: Search some page and do some action like open the editor or set a parent page. Do it from a different filter selected, like published and then draft...
- Set Parent Page Search: Search some page and select it as parent

This PR also should fix #10253 @rachelmcr
